source/VMelnalksnis.PaperlessDotNet/Documents/DocumentClient.cs

```diff
@@ -109,6 +110,23 @@ public async IAsyncEnumerable<Document<TFields>> GetAll<TFields>(int pageSize, [
 		return await GetCore<Document<TFields>>(id, cancellationToken).ConfigureAwait(false);
 	}
 
+
+	/// <inheritdoc />
+	public async Task<DocumentContent> Download(int id, bool original, CancellationToken cancellationToken = default)
+	{
+		var response = await _httpClient.GetAsync(original ? Routes.Documents.DownloadUri(id).ToString() :
```

Comment:
`ToString` is not needed; should pass `cancellationToken` and use `ConfigureAwait`
```suggestion
		var downloadUri = original ? Routes.Documents.DownloadUri(id) : Routes.Documents.DownloadOriginalUri(id);
		var response = await _httpClient.GetAsync(downloadUri, cancellationToken).ConfigureAwait(false);
```